fe/fe-core/src/main/java/com/starrocks/analysis/DescribeStmt.java

```diff
@@ -166,6 +166,11 @@ public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
                 ErrorReport.reportAnalysisException(ErrorCode.ERR_BAD_TABLE_ERROR, dbTableName.getTbl());
             }
 
+            if (table.getType() == TableType.HIVE) {
+                // Reuse the logic of `desc <table_name>` because hive external table doesn't support view.
+                isAllTables = false;
```

Comment:
I think it is better to use a local variable other than changing the member variable.

Reply:
The member variables here are only used as markers, and the effect should be controllable.